src/Raven.Studio/typescript/common/queryUtil.ts

```diff
@@ -39,12 +39,22 @@ class queryUtil {
         const fromPart = `from index ${escapedIndexName}`;
         let wherePart = `where ${escapedFieldName} == ${escapedValueName}`;
         
-        if (indexName.startsWith("Auto") && (value.startsWith("{") || value === "NULL_VALUE")) {
+        if (indexName.startsWith("Auto") && 
+            ((value.startsWith("{") && queryUtil.hasUpperCase(value)) || value === "NULL_VALUE")) {
```

Comment:
do we need `value.startsWith("{")` ?

Reply:
yes, no need to add 'exact' if term is not an object